swap_meet/vendor.py

```diff
@@ -1,2 +1,85 @@
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        self.inventory = inventory or []
```

Comment:
This is not syntax I had seen before and I intuitively didn't think it would work (I knew it did because all of your tests pass), but great job researching and finding this particular syntax!